contrib/syntax/textmate/Docker.tmbundle/Syntaxes/Dockerfile.tmLanguage

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
That very well might be! Can we confirm this somehow? I changed it to "Apple Computer" to align with the files in my "[source repo](https://github.com/asbjornenge/Docker.tmbundle)". If it should be "Apple" I'd be happy to modify the files in the source repo and squash another commit into this PR :wink: